tools/install.sh

```diff
@@ -42,20 +43,22 @@ fi
 # Get the OS type
 os_type=$(uname -s | tr '[:upper:]' '[:lower:]')
 
-# Get latest version of mole available
-	latest_version=$(curl --silent --location --max-time "${curl_timeout_seconds}" "https://api.github.com/repos/${repository}/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')
-if [ $? -ne 0 ]; then
-	echo -ne "\nThere was an error trying to check what is the latest version of mole.\nPlease try again later.\n"
-	exit 1
+if [ -z "$target_version" ]; then
+	# Get latest version of mole available
+	target_version=$(curl --silent --location --max-time "${curl_timeout_seconds}" "https://api.github.com/repos/${repository}/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')
+	if [ $? -ne 0 ]; then
+		echo -ne "\nThere was an error trying to check what is the latest version of mole.\nPlease try again later.\n"
+		exit 1
+	fi
 fi
 
-filename="mole${latest_version#v}.${os_type}-amd64.tar.gz"
-download_link="https://github.com/${repository}/releases/download/${latest_version}/${filename}"
+filename="mole${target_version#v}.${os_type}-amd64.tar.gz"
+download_link="https://github.com/${repository}/releases/download/${target_version}/${filename}"
```

Comment:
```suggestion
download_link="https://github.com/${repository}/releases/download/v${target_version}/${filename}"
```

There is missing `v` at the beginning of the tag version in the download url.

Reply:
Got it, I put it without the `v` expecting the whole version tag, but make more sense to have it there and expect the version number only 👍

Reply:
PR description updated.